setup.py

```diff
@@ -7,13 +7,13 @@
 data_dir = os.path.join(sysconfig.get_python_lib(), "hamster", "data")
 
 setup(
-    name='hamster-sqlite',
-    version='0.3',
-    description='Minimal dependency nicely abstracted sqlite backend of hamster time tracker - lets you connect to your hamster db and do stuff in python',
-    author='Toms Baugis',
-    author_email='toms.baugis@gmail.com',
-    url='https://github.com/projecthamster/hamster',
-    package_dir={'': 'src'},
-    py_modules=['hamster.storage', 'hamster.db', 'hamster.lib.__init__'],
-    data_files=[(data_dir, ['data/hamster.db'])],
+    name="hamster-sqlite",
```

Comment:
What's the point of replacing `'` with `"`? I thought both are equivalent in python, is that no longer true?